tests/testsuite/build_script.rs

```diff
@@ -5168,8 +5168,8 @@ fn wrong_output() {
         .with_stderr(
             "\
 [COMPILING] foo [..]
-error: invalid output in build script of `foo v0.0.1 ([ROOT]/foo)`: `cargo:example`
-Expected a line with `cargo:key=value` with an `=` character, but none was found.
+error: invalid output in build script of `foo v0.0.1 ([ROOT]/foo)`: `cargo::example`
```

Comment:
Wouldn't his line have changed in the original commit?  Were tests run on each commit?

Reply:
Now, every commit can pass the tests. You can see the GitHub stats.